### PR TITLE
Scope updates

### PR DESCRIPTION
**Pull request by ryanmelt**
Updates to make scopes work as desired.

### CODE REVIEW

File: openc3-cmd-tlm-api/app/controllers/scopes_controller.rb [hunk 1]
```diff
@@ -23,4 +23,19 @@ class ScopesController < ModelController
...
+
+  def create(update_model = false)
+    return unless authorization('superadmin')
+    super(update_model)
+  end
+
+  def destroy
+    return unless authorization('superadmin')
+    begin
+      result = OpenC3::ProcessManager.instance.spawn(["ruby", "/openc3/bin/openc3cli", "destroyscope", params[:id]], "scope_uninstall", params[:id], Time.now + 2.hours, scope: 'DEFAULT')
```
Comment (jmthomas) [inline openc3-cmd-tlm-api/app/controllers/scopes_controller.rb:35]: Seems weird to have to give a scope: 'DEFAULT' to the process to destroy scopes

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue [hunk 2]
```diff
@@ -255,6 +255,8 @@ export default {
         'openc3-tool-tablemanager',
         'openc3-tool-tlmgrapher',
         'openc3-tool-tlmviewer',
+        'openc3-enterprise-tool-base',
+        'openc3-enterprise-tool-admin',
```
Comment (jmthomas) [inline openc3-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue:259]: I don't think these should be here

**Review by jmthomas — CHANGES_REQUESTED** [review]

File: openc3-cmd-tlm-api/app/controllers/scopes_controller.rb (hunk 1 above)
Comment (ryanmelt) [reply openc3-cmd-tlm-api/app/controllers/scopes_controller.rb:35]: The DEFAULT scope is also going to be the kindof "global scope" that owns Scopes and Roles.

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue (hunk 2 above)
Comment (ryanmelt) [reply openc3-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue:259]: The alternative is copy this whole file into enterprise and then add in the differences, or come up with some sort of included subfile.  This is the simplest for now and we can cleanup later.

**Pull request merged**